README.md

```diff
@@ -44,36 +44,39 @@ Please use the following to reach members of the community:
 - Email: [info@rook.io](mailto:info@rook.io)
 
 ### Community Meeting
+
 A regular community meeting takes place every other [Tuesday at 9:00 AM PT (Pacific Time)](https://zoom.us/j/392602367).
 Convert to your [local timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
 
 Any changes to the meeting schedule will be added to the [agenda doc](https://docs.google.com/document/d/1exd8_IG6DkdvyA0eiTtL2z5K2Ra-y68VByUUgwP7I9A/edit?usp=sharing) and posted to [Slack #announcements](https://rook-io.slack.com/messages/C76LLCEE7/) and the [rook-dev mailing list](https://groups.google.com/forum/#!forum/rook-dev).
 
 Anyone who wants to discuss the direction of the project, design and implementation reviews, or general questions with the broader community is welcome and encouraged to join.
-* Meeting link: https://zoom.us/j/392602367
-* [Current agenda and past meeting notes](https://docs.google.com/document/d/1exd8_IG6DkdvyA0eiTtL2z5K2Ra-y68VByUUgwP7I9A/edit?usp=sharing)
-* [Past meeting recordings](https://www.youtube.com/playlist?list=PLP0uDo-ZFnQP6NAgJWAtR9jaRcgqyQKVy)
+
+- Meeting link: <https://zoom.us/j/392602367>
```

Comment:
Is linter recommending to put spaces between the bullet points? It seems better to me without the spaces, but it's a very minor thing

Reply:
@travisn if you look at this https://github.com/rook/rook/blob/a03c81855bd6deda95c51af423831355ee65a530/README.md , Isnt it looking good compared to without spaces ?

Reply:
@travisn its also consistent with other lists in the same README

Reply:
The spacing looks different from other lists. For example, under "Contacts", Slack, Forums, Twitter, Email don't have a space, neither do Alpha, Beta, Stable under the project status.

Reply:
@travisn PTAL on latest patch set. I have removed the spaces.

Reply:
sorry, i wasn't clear on the spaces. I was referring to the extra lines between the items. I'm not normally this picky, but it's for the main readme. :)

Reply:
@travisn nw, I have addressed it latest patch set. ptal :)